Source/WebKit/Platform/cocoa/CocoaWindow.h

```diff
@@ -27,8 +27,8 @@
 
 #if USE(APPKIT)
 OBJC_CLASS NSWindow;
-using CocoaWindow = NSWindow;
+typedef NSWindow CocoaWindow;
 #else
 OBJC_CLASS UIWindow;
-using CocoaWindow = UIWindow;
+typedef UIWindow CocoaWindow;
```

Comment:
I am surprised by this change, we've been replacing typedefs with `using` statements for a while now so this seems to be going in the opposite direction to the rest of the codebase.